code/modules/mob/living/carbon/human/species.dm

```diff
@@ -510,10 +510,9 @@ GLOBAL_LIST_EMPTY(features_by_species)
 		C.dna.blood_type = random_blood_type()
 	for(var/X in inherent_traits)
 		REMOVE_TRAIT(C, X, SPECIES_TRAIT)
-	for(var/obj/item/organ/external/organ in C.internal_organs)
-		if(organ.type in external_organs)
-			organ.Remove(C)
-			qdel(organ)
+	for(var/obj/item/organ/external/organ as anything in C.external_organs)
```

Comment:
hmm are you sure this is the same behaviour?

Reply:
It isn't, the previous one was completely non-functional as there are no external organs in internal_organs.